Dockerfile-cli

```diff
@@ -14,12 +14,14 @@ COPY src/php/utils/docker/ /usr/local/bin/
 
 # Install PHP extensions
 RUN set -x \
-    && apk add --no-cache wget \
+    && apk add --no-cache libsodium-dev wget \
```

Comment:
Is the addition of `libsodium` specific to `PHP 7.1`?

Reply:
Yes.

Reply:
Do you think it'd be worth it to find a way where we only install it on the `7.1` images? Now it's been added to all our PHP images.

Reply:
[As of 7.2 it's bundled with PHP by default, so we already have it in our images.](https://www.php.net/manual/en/sodium.installation.php)

Reply:
Might be nice to document this as a comment in the Dockerfile

Reply:
Added a short comment to the Dockerfiles